build/KoreBuild.ps1

```diff
@@ -119,15 +119,22 @@ if (!($env:Path.Split(';') -icontains $dotnetLocalInstallFolder))
 
 $makeFileProj = "$PSScriptRoot/KoreBuild.proj"
 $msbuildArtifactsDir = "$repoFolder/artifacts/msbuild"
-$msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.binlog"
 $msBuildResponseFile = "$msbuildArtifactsDir/msbuild.rsp"
 
+$msBuildLogArgument = ""
+
+if ($env:KOREBUILD_ENABLE_BINARY_LOG -eq "1")
+{
+    Write-Host "Enabling binary logging because KOREBUILD_ENABLE_BINARY_LOG = 1"
+    $msbuildLogFilePath = "$msbuildArtifactsDir/msbuild.binlog"
+    $msBuildLogArgument = "/bl:$msbuildLogFilePath"
```

Comment:
So sorry I've missed it, but `/bl` will cause the perf problems for you. Didn't we decide to use the `/logger` parameter instead?